scripts/build_mod_ic.py

```diff
@@ -359,6 +359,25 @@
         },
     },
 
+    # QFN-20-EP-MAX for MAX17503
+    "QFN-20-EP-MAX": {
+        "rows": 4,
+        "pins": 20,
+        "pin_pitch": 0.5,
+        "row_pitch": 3.85,
+        "pad_shape": (0.6, 0.3),
+        "ep_shape": (2.85, 2.85),
+        "ep_paste_shape": (1.1, 1.1, 0.2, 0.2),
```

Comment:
Probably worth putting vias on here? The datasheet specifically says to add thermal vias below the exposed pad. Probably one in the centre of each paste pad is the best you can do.